### PR TITLE
PP-19: Add forgot password pages

### PR DESCRIPTION
**Pull request by sanjay-techuz**

### CODE REVIEW

File: src/pages/Auth/ForgotPassword/ForgotPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    email: Yup.string().email('Invalid email').required('Email is Required')
+  });
+
+  const fPFormik = useFormik({
+    initialValues: {
+      email: ''
+    },
+    validationSchema: forgotPasswordValidationSchema,
+    onSubmit: (values) => {
+      setIsResetPassword(true);
```
Comment (ketanp-techuz) [inline src/pages/Auth/ForgotPassword/ForgotPassword.jsx:30]: As per the doc this is not the flow for rest password, once we add the email and if it's verified with our DB then will sent one email to user and user will redirected to the app and then they can see the reset password screen. for now you can just separate the both routes or you can add any identifier.

Comment (sanjay-techuz) [reply src/pages/Auth/ForgotPassword/ForgotPassword.jsx:30]: OK.then i create separate route for reset password. so we can redirect user's from email

**Comment by sanjay-techuz** [comment]
[Add forgot password pages](https://trello.com/c/PNPXxhAm/19-add-forgot-password-pages)

**Comment by sanjay-techuz** [comment]
[Add forgot password pages](https://trello.com/c/PNPXxhAm/19-add-forgot-password-pages)

File: src/pages/Auth/ResetPassword/ResetPassword.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  });
+
+  const rPFormik = useFormik({
+    initialValues: {
+      newPassword: '',
+      confirmPassword: ''
+    },
+    validationSchema: resetPasswordValidationSchema,
+    onSubmit: (values) => {
+      navigate('/');
```
Comment (ketanp-techuz) [inline src/pages/Auth/ResetPassword/ResetPassword.jsx:31]: First set the values and then call the navigate

**Review by ketanp-techuz — APPROVED** [review]

**Pull request merged**